RecoEgamma/EgammaElectronAlgos/src/GsfElectronAlgo.cc

```diff
@@ -1543,8 +1537,9 @@ void GsfElectronAlgo::createElectron(const gsfAlgoHelpers::HeavyObjectCache* hoc
   //====================================================
 
   reco::GsfElectron::IsolationVariables dr03, dr04 ;
-  dr03.tkSumPt = eventData_->tkIsolation03->getPtTracks(ele);
-  dr04.tkSumPt = eventData_->tkIsolation04->getPtTracks(ele);
+  dr03.tkSumPt = tkIsol03Calc_.calIsolPt(*ele->gsfTrack(),*eventData_->currentCtfTracks);
+  dr04.tkSumPt = tkIsol04Calc_.calIsolPt(*ele->gsfTrack(),*eventData_->currentCtfTracks);
```

Comment:
some odd [historical] naming choice: calIsol actually returns track sum

Reply:
calIsol returns a pair of nr tracks, track pt sum.  Yes this is for historical reasons. Do you want me to do anything about it? Perfectly happy to if you think its confusing.

Reply:
No strong preference for now or later.
Still, it would eventually be easier to see in the name that it does track isolation.
Maybe I missed something from the history why "cal" should mean "track"

Reply:
ah, now I see what you read and I understand!

calIsol = calculate isolation

not  caloimeter isolation

Hmm, I honestly hadnt noticed. The class is a track isolation calculator so the method names only refer to calculating the isolation and its assumed to be the track isolation. Hmm. Debating what to do about that, will have a think.

Reply:
OK to leave as is